core/gnupg/gnupg.install

```diff
@@ -4,7 +4,7 @@ post_install() {
 }
 
 post_upgrade() {
-  if [[ $(vercmp $2 2.1.13-1) = -1 ]]; then
+  if [ $(vercmp $2 2.1.13-1) = -1 ]; then
```

Comment:
Didn't we switch the install script to bin/sh anyways?

Reply:
@Amzo Yeah we did, hence my last comment where I pointed out that we made that switch a while back.

Reply:
It would be worth checking pacman configure args in the PKGBUILD to double check which shell script we use. I know we did switch to /bin/sh but I think we might have moved back to bash for the use of arrays. As for '[[' that is bash only.

Reply:
Pacman is using `/bin/sh`. If we wanted to use `[[`, couldn't we make a symbolic link from `/bin/[[` to `/bin/[`?

Reply:
We are indeed using `/bin/sh` currently for [pacman](https://github.com/PacBSD/abs/blob/master/core/pacman/PKGBUILD#L71) with no negitive impact on arrays (that I have seen). Only big downside is mixups like these where we were once using bash to parse the PKGBUILD and install files but are no longer.

Personally I would be okay with either just sticking to the bourne shell and only using the `[` syntax or switching back to bash and using the `[[` rather than mixing the two via a symlink.

Reply:
I'm not too concerned with [ or [[, but the problem I think may be when adding new users to the system, checking if users exist and removing them, look at git install script where we add the git shell to shells, and add a git users. It's messy in in /bin/sh compatibility, and I think we may be better off moving back to bash. However, the reason we switched to bin/sh was because if bash breaks during an upgrade say of freebsd-world were libraries were bumped, all install scripts are going to break until bash is fixed. So if you upgrade world bash breaks, scripts that add new users to the system essentially break thus breaking the package as well.

bin/sh should never break as it's always upgraded and part of world.

Reply:
Agreed, and as long as we make sure to rebuild/rev bump bash when we upgrade freebsd-world we should be okay. On the other hand IIRC `bash` itself has an install file so if such a breakage were to take place then unless the `post_install` or `post_upgrade` functions truly don't fire until after the install/upgrade is done we may be in for a a lot of suffering.

Maybe discussing this in this PR is not the best place, and we should move this either to IRC or to https://groups.google.com/forum/#!forum/pacbsd-dev-public